source/scripts/model/Dungeon.js

```diff
@@ -19,11 +20,36 @@ export class Dungeon {
         })
     }
 }
+/**
+ * Class creates a random dungeon.
+ * @todo Position the stairs better.
+ */
+export class RandomDungeon extends Dungeon {
+    constructor(dungeon) {
+        super(dungeon)
+
+        let gen = new Generator()
```

Comment:
So what are your thoughts on separating the dungeon generation from the dungeon object? Would you be opposed to merging them, putting the generation code inside the class? (So `this.spaces = this.generate(this.size)` as opposed to `this.spaces = gen.generate(this.size)`)